allennlp/models/simple_tagger.py

```diff
@@ -141,6 +135,16 @@ def tag(self, text_field: TextField) -> Dict[str, Any]:
         predictions = output_dict["class_probabilities"].data.squeeze(0)
         _, argmax = predictions.max(-1)
         indices = argmax.squeeze(1).numpy()
-        tags = [self.vocabulary.get_token_from_index(x, namespace="tags") for x in indices]
+        tags = [self.vocab.get_token_from_index(x, namespace="tags") for x in indices]
 
         return {"tags": tags, "class_probabilities": predictions.numpy()}
+
+    @classmethod
+    def from_params(cls, vocab: Vocabulary, params: Params) -> 'SimpleTagger':
+        hidden_size = params.pop("hidden_size", 200)
+        num_layers = params.pop("num_layers", 2)
+        token_embedder = TokenEmbedder.from_params(vocab, params.pop("token_embedder"))
+        return cls(vocab=vocab,
```

Comment:
My thought was that it could be helpful if you want to subclass `SimpleTagger`.  Thinking about it a bit more, though, I'm not sure you actually get any benefit here from that, because this method returns an instance of `cls`, which I believe is always guaranteed to be `SimpleTagger`.  Maybe @joelgrus has some comments on this?  I'd be fine with switching them to static methods if the class method really isn't giving us anything, but I'd recommend doing that everywhere in a single PR.